### ISSUE TITLE
Bytes range request will not response Cache-Control header

### ISSUE THREAD

**Issue by ssssssssssss** [jfhbrook/node-ecstatic#198]
Request headers:
```
Cache-Control:max-age=3600
Connection:keep-alive
Range:bytes=0-1540095
```
Response headers:
```
Accept-Ranges:bytes
Connection:keep-alive
Content-Length:1540096
Content-Range:bytes 0-1540095/129882056
Content-Type:video/mp4; charset=utf-8
Date:Thu, 18 Jul 2017 03:18:58 GMT
server:ecstatic-2.2.1
```

The default 3600s setting is not present.

**Comment by jfhbrook** [comment]
Can you elaborate? What's the expected behavior, vs the actual behavior? Act like I'm dumb and haven't thought about this library for a few months.

**Comment by jfhbrook** [comment]
It looks like you're showing me the actual response; but what's the expected response?

**Comment by ssssssssssss** [comment]
Expected additional response headers:
```
cache-control: max-age=3600
etag:W/"1618409-2346993-"2017-07-18T02:15:39.000Z""
last-modified:Thu, 18 Jul 2017 02:15:39 GMT
```
I tested against latest Chrome 59, including the listed 3 headers will make Chrome cache the content fragment correctly.

**Comment by jessetane** [comment]
Took a couple tries but I think I fixed this with #204, @ssssssssssss could you take a look?

**Comment by ssssssssssss** [comment]
Thanks, it looks good. 👍
